api/README.md

```diff
@@ -10,8 +10,8 @@ Before you can start packit you need to run `./scripts/run-dependencies` from th
 to start database and `outpack_server` instances.
 
 ## Starting App
-Run up app on command line from the project root directory
```

Comment:
my fault i forgot to add.. but should mention about also adding in GITHUB_CLIENT_ID & GITHUB_CLIENT_SECRET..... otherwise this alll wont work